### PR TITLE
openshift-sdn to ovn-kubernetes migration testing feedback

### PR DESCRIPTION
**Pull request by dlbewley**
Suggested updates for openshift-sdn to ovn-kubernetes migration docs.

### CODE REVIEW

**Comment by jboxman** [comment]
@dlbewley thanks, I'll take a look!

**Comment by dlbewley** [comment]
if i had a nickel for every time i've mistyped kubernetes..., i'd be less embarrased

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 1]
```diff
@@ -43,10 +43,10 @@ $ oc patch Network.config.openshift.io cluster \
   --type='merge' --patch '{ "spec": { "networkType": "OVNKubernetes" } }'
 ----
 
-. To confirm the migration disabled the OpenShift SDN default CNI network provider, enter the following command. It might take several moments for all the OpenShift SDN Pods to stop.
+. To confirm the migration disabled the OpenShift SDN default CNI network provider and removed all OpenShift SDN Pods, enter the following command. It might take several moments for all the OpenShift SDN Pods to stop.
 +
 ----
-$ oc get pod -n openshift-sdn --watch
+$ watch oc get pod -n openshift-sdn
```
Comment (jboxman) [inline modules/nw-ovn-kubernetes-migration.adoc:49]: @dlbewley, what makes this preferable than `--watch`?

Comment (dlbewley) [reply modules/nw-ovn-kubernetes-migration.adoc:49]: The condition the user wants to see is that all the pods are gone. Using `--watch` simply emits a stream of changes . It is much more obvious when the pods are gone  with `watch <cmd>`, particularly if there are many pods (nodes). 

If watch can be assumed to be broadly available I think the output is much more clear.

**Comment by jboxman** [comment]
Hi @huiran0826,

Do these updates make sense?

Thanks!

**Comment by huiran0826** [comment]
lgtm

File: modules/nw-ovn-kubernetes-rollback.adoc [hunk 2]
```diff
@@ -39,10 +39,10 @@ $ oc patch Network.config.openshift.io cluster \
 $ oc edit Network.config.openshift.io cluster
 ----
 
-. To confirm that the migration disabled the OVN-Kubernetes default CNI network provider, enter the following command. It might take several moments for all the OVN-Kubernetes Pods to stop.
+. To confirm that the migration disabled the OVN-Kubernetes default CNI network provider and removed all the OVN-Kubernetes pods, enter the following command. It might take several moments for all the OVN-Kubernetes Pods to stop.
```
Comment (codyhoag) [inline modules/nw-ovn-kubernetes-rollback.adoc:42]: s/pods/Pods

File: modules/nw-ovn-kubernetes-migration.adoc [hunk 3]
```diff
@@ -26,7 +26,7 @@ Perform the migration only when an interruption in service is acceptable.
 . To backup the configuration for the cluster network, enter the following command:
 +
 ----
-$ oc get Network.config.openshift.io cluster -o yaml
+$ oc get Network.config.openshift.io cluster -o yaml  > cluster-openshift-sdn.yaml
```
Comment (codyhoag) [inline modules/nw-ovn-kubernetes-migration.adoc:29]: There's an extra space before `>`

**Review by codyhoag — COMMENTED** [review]
Two small suggestions. Otherwise, LGTM!

**Comment by jboxman** [comment]
@dlbewley, if you can fix the two issues above we're good to go with this.

Thanks!

**Comment by jboxman** [comment]
@dlbewley, squashed and merged, thanks!

**Pull request merged**